src/transmitter/index.ts

```diff
@@ -196,35 +196,30 @@ export async function sendWorkloadEventsPolicy(
   }
 }
 
-export async function deleteWorkload(
-  payload: IDeleteWorkloadPayload,
-): Promise<void> {
+export async function deleteWorkload(params: string): Promise<void> {
   try {
     const request: KubernetesUpstreamRequest = {
       method: 'delete',
-      url: `${upstreamUrl}/api/v1/workload`,
-      payload,
+      url: `${upstreamUrl}/api/v1/workload/${params}`,
+      payload: {} as IDeleteWorkloadPayload,
```

Comment:
is there a way to get rid of this?

Reply:
Maybe on KubernetesUpstreamRequest type, the payload could be undefined?

Reply:
Are we using this new delete endpoint for all requests from now on? Or are we still using the old delete endpoint for dev/prod 16?